esmf-semantic-aspect-meta-model/src/main/resources/samm/characteristic/2.0.0/characteristic-shapes.ttl

```diff
@@ -783,3 +783,27 @@ samm-c:FixedPointConstraintShape
       sh:name "integer" ;
       sh:description "The number of integral digits for a fixed point number." ;
    ] .
+
+samm-c:TimeSeriesShape
+   a sh:NodeShape ;
+   sh:targetClass samm-c:TimeSeries ;
+   sh:property [
+      sh:path samm:dataType ;
+      sh:minCount 0 ;
+      sh:maxCount 1 ;
+      sh:name "dataType" ;
+      sh:description "This constraint for TimeSeries type and for checking extending this type from TimeSeriesEntity." ;
+      sh:sparql [
+         a sh:SPARQLConstraint ;
+         sh:message "TimeSeries Entry to be extended from TimeSeriesEntity. You can use this structure for resolve it: 'samm:extends samm-e:TimeSeriesEntity ;'" ;
+         sh:prefixes samm:prefixDeclarations ;
+         sh:select """
+            select $this ?definedDataType
+            where {
+               $this rdf:type samm-c:TimeSeries .
```

Comment:
This constraint is not needed, in the context of the validator it's function is fulfilled by the sh:targetClass property.

Reply:
Thank you for this comment, I am going to remove it!